python/tests/CMakeLists.txt

```diff
@@ -57,4 +62,9 @@ if (NOT ${PYDANTIC_result} EQUAL 0)
   return()
 endif()
 
+if (NOT ${IQM_CLIENT_result} EQUAL 0)
```

Comment:
This will skip all remote mock server tests if the IQM client is not installed, which is unnecessary. Please remove the change in this file, along with the edit of the devdeps.Dockerfile, and instead simply add a pytest.importorskip to the IQM mock server tests.